lib/archethic/utils/regression/benchmarks/tps.ex

```diff
@@ -0,0 +1,98 @@
+defmodule ArchEthic.Utils.Regression.Benchmark.TPS do
+  @moduledoc """
+  Module for regession testing the paralleing processing of transactions and
+  benchmarking the parallel transaction processing capability of a node
+  """
+
+  require Logger
+
+  alias ArchEthic.Utils.Regression.Benchmark
+  alias ArchEthic.Utils.Regression.Playbook
+  alias ArchEthic.Crypto
+  alias ArchEthic.Utils.Regression.Benchmarks.Helpers.TPSHelper
+
+  alias ArchEthic.TransactionChain.TransactionData
+  alias ArchEthic.TransactionChain.TransactionData.Ledger
+  alias ArchEthic.TransactionChain.TransactionData.UCOLedger
+  alias ArchEthic.TransactionChain.TransactionData.UCOLedger.Transfer, as: UCOTransfer
+
+  @behaviour Benchmark
+
+  def plan([host | _nodes], _opts) do
+    port = Application.get_env(:archethic, ArchEthic.P2P.Listener)[:port]
+
+    {alice, bob} = preliminaries()
+
+    TPSHelper.withdraw_uco_from_testnet_endpoint(alice.address)
```

Comment:
As mentioned earlier, Archethic transaction  has the capability to do bulk transfers . So with a single transaction you can target multiple recipients.

Reply:
It's on the UCO ledger, with multiple transfers